samples/tab-deeplink/csharp/DeepLinkBot/DeeplinkHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Web;
+
+namespace Microsoft.BotBuilderSamples.Bots
+{
+    public class DeeplinkHelper
+    {
+        public static IConfiguration _configuration;
+        public DeeplinkHelper(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        static Dictionary<string, string> task1Values = new Dictionary<string, string>
+            {
+                {"subEntityId","topic1" }
```

Comment:
Have two methods to fetch deeplink. 

string GetDeepLinkToPersonalTab()
string  GetDeepLinkToChannelTab()